_includes/manuals/1.15/4.3.8_smartproxy_realm.md

```diff
@@ -6,11 +6,13 @@ Activate the realm management module within the Smart Proxy instance.  This mana
 :realm_provider: freeipa
 </pre>
 
-`freeipa` is the only provider currently available.
+We have a provider for `freeipa` available. The configuration for each provider should be
```

Comment:
1. Avoid pronouns - perhaps use the same format (a bullet point list) as other sections (DNS, DHCP) with provider support.
1. It's the `realm_freeipa` provider, or it's _for FreeIPA_, or both. This should probably read: "`realm_freeipa` - host object management in FreeIPA"